onnxconverter_common/optimizer.py

```diff
@@ -700,7 +700,14 @@ def apply(self, node_list):
         var = numpy_helper.to_array(self.end_p.get_precedence_by_idx(4).tensors[0])
         epsilon = self.end_p.get_attribute('epsilon', 1.0e-5)
         adjusted_scale = scale / np.sqrt(var + epsilon)
-        conv_weight = conv_ori_weight * adjusted_scale[:, None, None, None]
+        if len(conv_ori_weight.shape) == 4:
+            conv_weight = conv_ori_weight * adjusted_scale[:, None, None, None]
+        elif len(conv_ori_weight.shape) == 3:
+            conv_weight = conv_ori_weight * adjusted_scale[:, None, None]
+        elif len(conv_ori_weight.shape) == 2:
+            conv_weight = conv_ori_weight * adjusted_scale[:, None]
+        else:
+            return None, False
```

Comment:
Is it a good idea to throw an error in this else case?

Reply:
For this case, we just skip this fusion optimization